src/components/Header.tsx

```diff
@@ -11,6 +11,13 @@ const Header = () => (
     </div>
     <div className="flex flex-row items-center space-x-8 text-xs tracking-widest">
       <a className="opacity-50">DOCS</a>
+      <a
```

Comment:
Same here, actually maybe reducing the space between links on mobile would help (space-x-4 md:space-x-8) but need to make sure it doesn't look too compact. Worst case, the cap.uly.dev client redirects to the governance so I don't think it's a big issue

Reply:
ok, reversed